src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
```suggestion
  a: number;
  b: number;
  c: number;
```
yeap, remove empty lines